src/Api/AdminConsole/Controllers/OrganizationUsersController.cs

```diff
@@ -682,4 +694,15 @@ private async Task<ListResponseModel<OrganizationUserBulkResponseModel>> Restore
         return new ListResponseModel<OrganizationUserBulkResponseModel>(result.Select(r =>
             new OrganizationUserBulkResponseModel(r.Item1.Id, r.Item2)));
     }
+
+    private async Task<IDictionary<Guid, bool>> GetManagedByOrganizationStatusAsync(Guid orgId, IEnumerable<Guid> userIds)
+    {
+        if (!_featureService.IsEnabled(FeatureFlagKeys.AccountDeprovisioning))
+        {
+            return userIds.ToDictionary(kvp => kvp, kvp => false);
+        }
+
+        var organizationsManagingUsers = await _getOrganizationUsersManagementStatusQuery.GetUsersOrganizationManagementStatusAsync(orgId, userIds);
+        return organizationsManagingUsers.ToDictionary(kvp => kvp.Key, kvp => kvp.Value);
```

Comment:
It looks like `ToDictionary` is redundant, the query already returns an `IDictionary` in this format.